crates/ruff_python_parser/tests/snapshots/invalid_syntax@invalid_string_prefix.py.snap

```diff
@@ -0,0 +1,120 @@
+---
+source: crates/ruff_python_parser/tests/fixtures.rs
+input_file: crates/ruff_python_parser/resources/inline/err/invalid_string_prefix.py
+---
+## AST
+
+```
+Module(
+    ModModule {
+        node_index: AtomicNodeIndex(..),
+        range: 0..40,
+        body: [
+            Expr(
+                StmtExpr {
+                    node_index: AtomicNodeIndex(..),
+                    range: 0..7,
+                    value: StringLiteral(
+                        ExprStringLiteral {
+                            node_index: AtomicNodeIndex(..),
+                            range: 0..7,
+                            value: StringLiteralValue {
+                                inner: Single(
+                                    StringLiteral {
+                                        range: 0..7,
+                                        node_index: AtomicNodeIndex(..),
+                                        value: "hey",
+                                        flags: StringLiteralFlags {
+                                            quote_style: Double,
+                                            prefix: Empty,
+                                            triple_quoted: false,
+                                        },
+                                    },
+                                ),
+                            },
+                        },
+                    ),
+                },
+            ),
+            Expr(
+                StmtExpr {
+                    node_index: AtomicNodeIndex(..),
+                    range: 8..17,
+                    value: StringLiteral(
+                        ExprStringLiteral {
+                            node_index: AtomicNodeIndex(..),
+                            range: 8..17,
+                            value: StringLiteralValue {
+                                inner: Single(
+                                    StringLiteral {
+                                        range: 8..17,
+                                        node_index: AtomicNodeIndex(..),
+                                        value: "there",
+                                        flags: StringLiteralFlags {
+                                            quote_style: Double,
+                                            prefix: Empty,
```

Comment:
`Empty` seems suboptimal. I think we should try to preserve the most important prefix (e.g. if `f` is present, favor it over `u`)

Reply:
Not sure for `ft` but I would ignore the binary prefix for the second case. 

My concern with the current solution is that ignoring both prefixes is the least correct. Ignoring just one would be closer to what the user intended.